design-system/react/src/components/Avatar/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const getBackgroundColor = (
+  backgroundColor?: string,
+  unique?: string
+): string | undefined => {
+  if (backgroundColor === 'fuel') return '#58c09b';
+  if (backgroundColor !== 'random') return backgroundColor;
+  if (!unique) return unique;
+
```

Comment:
```suggestion
  backgroundColor?: 'fuel' | 'random' | string
): string | undefined => {
  if (backgroundColor === 'fuel') return '#58c09b';
  if (backgroundColor !== 'random') return backgroundColor;

```

Reply:
Can we remove the unique prop?

Reply:
`unique` is being directly used to generate the color. It is basically the wallet address or any unique hash informed via `hash` prop in `AvatarGenerated` component.

Reply:
maybe just change for hash to follow the same name?